include/forcing/GridDataSelector.hpp

```diff
@@ -0,0 +1,300 @@
+#pragma once
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#include <boost/core/span.hpp>
+
+#include <geojson/JSONGeometry.hpp>
+
+struct Cell {
+    std::uint64_t x = 0;
+    std::uint64_t y = 0;
+    std::uint64_t z = 0;
+    double    value = NAN;
+
+    friend inline bool operator<(const Cell& a, const Cell& b) {
+        // TODO: Not really meaningful
+        return a.x < b.x || a.y < b.y || a.z < b.z;
+    }
+};
+
+struct Extent {
```

Comment:
Thinking ahead to where this is included alongside other sorts of selectors, maybe this should be renamed 
```suggestion
struct BoxExtent {
```